### PR TITLE
PID: Implement Bayesian PID response

### PR DESCRIPTION
**Pull request by njacazio**

### CODE REVIEW

File: Analysis/DataModel/include/AnalysisDataModel/PID/PIDResponse.h [hunk 1]
```diff
@@ -281,6 +282,36 @@ DECLARE_SOA_TABLE(pidTPCAl, "AOD", "pidTPCAl", //! Table of the TPC response wit
 
 #undef DEFINE_UNWRAP_NSIGMA_COLUMN
 
+namespace pidbayes
+{
+// Bayes probability
+DECLARE_SOA_COLUMN(BayesEl, bayesEl, float);                        //! Bayesian probability of having a Electron
```
Comment (jgrosseo) [inline Analysis/DataModel/include/AnalysisDataModel/PID/PIDResponse.h]: Could we consider also here a reduced format (similar to the binned nsigma)?

Comment (njacazio) [reply Analysis/DataModel/include/AnalysisDataModel/PID/PIDResponse.h]: Hi, sure! Good idea! (Thanks for already having a look btw!)

**Comment by njacazio** [comment]
If we merge https://github.com/AliceO2Group/AliceO2/pull/6678 before we can simplify a bit checking the code

**Comment by jgrosseo** [comment]
@njacazio Do you have to update this one now after the other is merged?

**Review by jgrosseo — APPROVED** [review]

**Pull request merged**